### PR TITLE
Add Login urls, views, tests

### PR DESCRIPTION
**Pull request by melvynsngg**
To add Logins to the app

Add a login app

This would be independent of the rest of the app unless otherwise required

### CODE REVIEW

File: initialize_db.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Initialize users
+for user in USERS:
+	get_user = User.objects.filter(username=user['username'])
+	if not (len(get_user) == 0):
+		continue
+	cur_user = User.objects.create_user(username=user['username'], email=user['email'], password=user['password'])
+	groups_str = user['groups']
+	success, groups = to_groups(groups_str)
+	if not success:
+		print groups
```
Comment (chinjieh) [inline initialize_db.py]: Does this actually print to the console? I remember heroku 'sucks' up `print` statements. As in all `print` statements will not be shown until the server is shutdown.

That's why I created the `logger` configuration, can reference `IncidentLocation`. I think `logger.debug(_)` prints to console using that.

Comment (melvynsngg) [reply initialize_db.py]: Done

Comment (chinjieh) [reply initialize_db.py]: Oh that was fast, okay

**Review by chinjieh — APPROVED** [review]

**Comment by chinjieh** [comment]
@sngguojie Oh normally the branch can be deleted also. I helped you delete in this case.

**Pull request merged**